docs/source/extensions/datamodules.rst

```diff
@@ -292,76 +299,68 @@ Returns a special dataloader for inference. This is the dataloader that the Trai
 
     class MNISTDataModule(pl.LightningDataModule):
         def predict_dataloader(self):
-            return DataLoader(self.mnist_test, batch_size=64)
+            return DataLoader(self.mnist_predict, batch_size=64)
 
 
 transfer_batch_to_device
 ^^^^^^^^^^^^^^^^^^^^^^^^
-Override to define how you want to move an arbitrary batch to a device.
```

Comment:
I preferred the previous way of separating API docs from user guide.

Reply:
won't this one be better? user will get the API arguments + the example was just repeated here and already present in the API docs.